src/ast/mod.rs

```diff
@@ -551,12 +551,34 @@ pub enum Statement {
     Rollback { chain: bool },
     /// CREATE SCHEMA
     CreateSchema { schema_name: ObjectName },
-
     /// ASSERT <condition> [AS <message>]
     Assert {
         condition: Expr,
         message: Option<Expr>,
     },
+    /// DEALLOCATE [ PREPARE ] { name | ALL }
```

Comment:
This needs to be enclosed in backticks (as well as the `ASSERT` comment above -- sorry about missing that!), so that `cargo doc` output is properly formatted. Same below.